Library/Homebrew/test/cmd/info_spec.rb

```diff
@@ -18,6 +18,23 @@
       .and not_to_output.to_stderr
       .and be_a_success
   end
+
+  it "looks for a Cask with the same name if a given Formula does not exist", :needs_network do
+    setup_remote_tap "homebrew/cask"
+
+    expect { brew "info", "firefox" }
+      .to output(%r{Error:\sNo\savailable\sformula\swith\sthe\sname\s"firefox"\s\n
```

Comment:
Replace `\s` with a space. Can include the line below but don't need to include every other line. Also, this output should probably note that this is the output for a cask.

Reply:
I don't think I can replace `\s` with literal whitespace when I'm using free-spacing mode on the regular expression. I'm using free spacing mode to break the regex up across multiple lines for readability. But I think escaping the whitespace like `\ ` works if that's preferable to the `\s` pattern.

Do you have a preference for how to indicate the output is for a cask? Maybe just something like `Found the following cask:` to stdout?